Firmware/ControladorRobo/control.cpp

```diff
@@ -0,0 +1,51 @@
+#include "control.h"
+
+void controllersSetup()
+{
+    //Interrupcao do TIMER1, aka periodo de amostragem para calculo do controle de velocidade
+    TCCR1A = 0;
+    TCCR1B = 0;
+    TCNT1  = 0;
+    TCCR1A = _BV(COM0A1) | _BV(COM0B1) | _BV(WGM00);
+    TCCR1B = _BV(CS00);
+    //interrupt frequency (Hz) = (Arduino clock speed 16,000,000Hz) / (prescaler * (compare match register + 1))
+    OCR1A = 311; // compare match register for 10us delay
+    TCCR1B |= (1 << WGM12); // CTC mode
+    //  TCCR1B |= (1 << CS31); // 8 prescaler
+    TCCR1B |= (1 << CS10);
+    TCCR1B |= (0 << CS11);
+    TCCR1B |= (1 << CS12);
+    TIMSK1 |= (1 << OCIE1A); // enable timer compare interrupt
+}
```

Comment:
I Think that controllerSetup need to be a singleton that will be parent of Control,
in that case the setup will only run once and it will be integrated in Control class.